job_bundles/README.md

```diff
@@ -0,0 +1,308 @@
+# AWS Deadline Cloud Job Bundles
+
+Job Bundles are one of the tools that you can use to define your Jobs for Amazon Deadline Cloud. They encapsulate
+an [Open Job Description Job Template](https://github.com/OpenJobDescription/openjd-specifications/wiki) into a directory
+with additional information such the files and directories that your Jobs need for
+Deadline Cloud's [Job Attachments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html)
+feature. The [Deadline Cloud CLI](https://github.com/aws-deadline/deadline-cloud) provides ways for you to use a Job Bundle
+to submit Jobs to your Deadline Cloud Queues to be run.
+
+We recommend developing your own Job Bundle(s), either by-hand or programatically, for your custom Job submission needs. 
```

Comment:
'Recommend' makes it  sound like there is an alternative

Reply:
There is. You could develop just a Job Template and use the CreateJob API directly. I can clarify that point.

Reply:
I think this paragraph would be clearer if it pointed out what bundles give you compared to job templates instead of giving a blanket recommendation. Something in the spirit of, "Job bundles build on job templates to deliver additional features like job attachments and a GUI submitter." Someone not needing these features (e.g. CMF plus in house job submission GUI) might be better off using CreateJob + a template.

Reply:
Good suggestion, Stephen